components/core/header/nav-bar/nav-bar-items.js

```diff
@@ -11,6 +11,7 @@ export default Object.freeze({
         { i18nKey: 'talk', value: '/speaking/talk' },
         { i18nKey: 'tutorial', value: '/speaking/tutorial' },
         { i18nKey: 'recording', value: '/speaking/recording' },
+        { i18nKey: 'post session', value: '/speaking/post-session' },
```

Comment:
"poster session" might be more accurate. Remember to add the item at /components/core/header/nav-bar/NavBar.i18n.js too